1_fetch/src/cross_validation_functions.R

```diff
@@ -0,0 +1,21 @@
+
+get_nested_gages<-function(gagesii,nav_distance_km){
```

Comment:
Can you move .R file with this function to the `3_` directory? `git mv` can be used to move while git tracking the move

Reply:
Oh wait, it might already be there? I see the same .R file name in 4_

Reply:
Did you `git rm` the 1_fetch version of this file? I still see it in my files after pulling the latest commits.